src/XIVLauncher/App.xaml.cs

```diff
@@ -59,6 +60,12 @@ public class CmdLineOptions
             [CommandLine.Option("clientlang", Required = false, HelpText = "Client language to use.")]
             public ClientLanguage? ClientLanguage { get; set; }
 
+            [CommandLine.Option("exportconfig", Required = false, HelpText = "Export a zip of all config")]
```

Comment:
Are these necessary? What's your use case?

Reply:
Not strictly needed. I just like having command args for any of the file-based features. Makes it easy for testing when I want to specify custom appdata paths to certify that it unpacks correctly.